### ISSUE TITLE
Texts are not fully recognized

### ISSUE THREAD

**Issue by arashgmn** [itagagaki/DeepL-Selected-Text#2]
Hello Itagaki,

It's been a while after the new release that I noted that text selection and hitting the "Translate with DeepL" doesn't work properly. It sends either a space character (" ") to the *translate from* box, or the selected text partially (not sure if it helps but I observed that the text will break after the first occurrence of special characters; what is before that is sent for translation but the rest omitted).

I'd be happy to send you extra information that may help to find the reason. Although I might need your help to retrieve logging files or of that kind.

**My system info**

- OS: Ubuntu 18.04.5 LTS
- Thunderbird: 78.8.1 (64-bit)

**Comment by itagagaki** [comment]
Hi,

Thank you for the report. I'm not sure when it stopped working as expected. I'll look into it when I get some free time.

**Comment by itagagaki** [comment]
I found a work around: on the add-on options page, fix `Source` and `Target` to something other than `auto`.

#### How to open the options page
1. Click the menu button and click Add-ons, or select Tools > Add-ons from the menu bar.
2. Click the three-dot button on "DeepL selected text" and select Options, or click "DeepL selected text" and click Options.

I will try to improve `auto` to work as before.

**Comment by arashgmn** [comment]
Thanks for your prompt response! 

I tweaked a bit the source and target languages as well as open in a new tab/window option. The latter doesn't change anything. But once the source language is set `auto`, independent of the state of the `target` language, *right-click + translate with DeepL* passes only one space character ` ` for translation. If the source is set to something rather than `auto`, the text will be copied until the first special character (`#,|,/`, but not `:()` etc.).

So, in short, the special character problem persists for me.

**Comment by itagagaki** [comment]
Oh, sure. I'll look into that issue again when I have more time. Thanks!

**Comment by itagagaki** [comment]
I have resolved this issue. Version 1.3.0 is now available for download at https://addons.thunderbird.net/addon/deepl-selected-text/ . Please try it.
Thanks!

**Comment by arashgmn** [comment]
It works like a charm! Thanks for the great add-on.
